l10n_it_fatturapa_export_zip/wizard/export_fatturapa.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+
+import base64
+import io
+import zipfile
+from datetime import datetime
+from odoo import models, api, fields, _
+from odoo.exceptions import UserError
+
+
+class WizardAccountInvoiceExport(models.TransientModel):
+    _name = "wizard.fatturapa.export"
+
+    @api.model
+    def _default_name(self):
+        return "%s_%s" % (
+            _("E-invoice-export"), datetime.now().strftime('%Y%m%d%H%M'))
+
+    data = fields.Binary("File", readonly=True)
+    name = fields.Char('Filename', default=_default_name)
+
+    @api.multi
+    def export_zip(self):
+        attachments = []
+        for att in self.env[self._context['active_model']].browse(
+                self._context['active_ids']):
+            attachments += [att]
+        for att in attachments:
+            if att.exported_zip:
+                raise UserError(_(
+                    "Attachment %s already exported. Remove ZIP file first"
+                ) % att.name)
+            if not att.datas or not att.datas_fname:
+                raise UserError(
+                    _("Attachment %s does not have XML file") % att.name)
+
+        fp = io.BytesIO()
+        with zipfile.ZipFile(fp, mode="w") as zf:
+            for att in attachments:
+                zf.writestr(att.datas_fname, base64.b64decode(att.datas))
+        fp.seek(0)
+        data = fp.read()
+        export_report_name = _('E-Invoices XML')
```

Comment:
You can get this from the method for `name`'s default instead of creating a new one